tests/transformers/test_modeling.py

```diff
@@ -12,13 +12,58 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import unittest
-from paddlenlp.transformers import TinyBertModel
+import shutil
+from tempfile import TemporaryDirectory
+from tests.testing_utils import slow
+from multiprocessing import Pool
+from paddlenlp.transformers import TinyBertModel, AlbertModel
+from paddlenlp.utils.env import PPNLP_HOME
+
+
+def download_albert_model(model_name: str):
+    """set the global method: multiprocessing can not pickle local method
+
+    Args:
+        model_name (str): the model name
+    """
+    model = AlbertModel.from_pretrained(model_name, load_state_as_np=True)
+    # free the model resource
+    del model
+    return True
 
 
 class TestModeling(unittest.TestCase):
     """Test PretrainedModel single time, not in Transformer models"""
 
+    @slow
     def test_from_pretrained_with_load_as_state_np_params(self):
         """init model with `load_state_as_np` params"""
         TinyBertModel.from_pretrained("tinybert-4l-312d", load_state_as_np=True)
+
+    @slow
+    def test_multiprocess_downloading(self):
+        num_process, num_iter = 10, 20
+        small_model_path = "https://paddlenlp.bj.bcebos.com/models/community/__small_models__/bert-base-uncased/model_state.pdparams"
+
+        from paddlenlp.transformers.model_utils import get_path_from_url
+        with TemporaryDirectory() as tempdir:
+
+            with Pool(num_process) as pool:
+                pool.starmap(get_path_from_url, [(small_model_path, tempdir)
+                                                 for _ in range(num_iter)])
```

Comment:
what are we testing here?
This downloads the model `num_iter` times under `num_process` different processes. 
Are we trying to speedup the download of single big model file with multiprocess? Multiprocess can only help if the file can be broken into chunks, which i don't think is the case here.

Reply:
`num_process` is the number of process in ProcessPool, and the `num_iter` is the number of total process to download file.

this test code is to simulate the solution of distributed traning. Do you have any ideas about this code block ?

Reply:
OK. Would you add a comment to the code to document the reason for this test?